README.md

```diff
@@ -48,6 +48,17 @@ While the WebExtension:
 - Serves the survey to collect information from the user.
 
 
+### Survey
+
+Because extensions cannot run content scripts on other extensions' hosted pages, you cannot use the [React Developer Tools](https://addons.mozilla.org/firefox/addon/react-devtools/) to debug the survey in context of the add-on. To work around this, a command is included to run the survey from a development server:
```

Comment:
This works for me. But, just curious: Have you seen [Storybook](https://getstorybook.io/) for React? IMO, seems like an interesting way to host components in a web page with sample data. Been handy for fiddling with UI & CSS for Snooze Tabs outside the Web Extension context